inflection/src/inflection/dictionary/DictionaryMetaData.cpp

```diff
@@ -116,12 +115,12 @@ ::std::u16string* DictionaryMetaData::transform(::std::u16string* dest, std::u16
 int64_t* DictionaryMetaData::getCombinedBinaryType(int64_t* result, std::u16string_view word) const
 {
     *npc(result) = 0;
-    auto combinedType = npc(dictionary)->getWordType(word);
+    auto combinedType = dictionary->getWordType(word);
```

Comment:
If you're going to remove the null pointer checks like this, then the check needs to move to the constructor when the dictionary is assigned. It should never be null. Once the constructor has the null pointer check, I'll approve this pull request.

Reply:
PTAL